crates/wasm-pkg-loader/src/config.rs

```diff
@@ -44,6 +44,10 @@ impl ClientConfig {
         self
     }
 
+    pub fn get_default_registry(&self) -> Option<&str> {
```

Comment:
`get_` is discouraged: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
Yeah, there was a name conflict. I'll refactor the other names to have `set_` prefix.

Reply:
Refactored, let me know if that is OK now.